bio-jessica-gallinger

```diff
@@ -0,0 +1,14 @@
++---
```

Comment:
Hi @gallingerj,
The content looks good, but I don't understand the double '+' at the start of each line - can you please edit to remove those?
Thanks
@gvwilson